.slack/hooks.json

```diff
@@ -0,0 +1,6 @@
+{
+  "hooks": {
+    "get-hooks": "npx -q --no-install -p @slack/cli-hooks slack-cli-get-hooks",
+    "start": "npm run dev"
```

Comment:
Does the CLI properly handle a case where the hooks are predefined by the template?

Reply:
Having the CLI preferring the dev NPM script and falling back to the `start` command makes sense 💯 

Do we plan to also introduce an easy way for developers to override this? Maybe we can follow the principle of high configurability with good defaults